pf9/cluster/cluster_attach.py

```diff
@@ -124,8 +124,10 @@ def get_uuids(self, host_ips):
             if host_uuid is not None:
                 host_uuids.append(host_uuid)
             else:
-                logger.info("Unable to find host with IP:{}, please try again or run prep-node first".format(host_ip))
-                click.echo("Unable to find host with IP:{}, please try again or run prep-node first".format(host_ip))
+                error_msg = "Unable to find host with IP:{}, please try again or run prep-node first".format(host_ip)
```

Comment:
Isn't it better to handle the error tracking in the call site of this method? Right now it appears that the exception thrown is not caught at the call site in attach_cluster().

Reply:
Maybe not, I see that every invocation of attach_cluster() already handling CLIException so it will just get propagated all the way up, and the top level function can handle accordingly, Also if you look below in attach_cluster() function, it looks they are also raise exceptions expecting the caller to handle it.